docs/blog/2023-06-08-thanos-at-aiven.md

```diff
@@ -16,56 +16,130 @@ Aiven is a cloud-native data infrastructure platform that provides fully managed
 
-The community has been critical for us. One example we ran into around this portion of the implementation which we received some help from the other users and maintainers on the CNCF Slack. We also ran into some other issues which would create replication loops causing a crash as well. We addressed this issue by moving to a routing-ingesting receiver topology as suggested by the community.
+The community has been critical for us. One example we ran into around this portion of the implementation which we received some help from the other users and maintainers on the [CNCF Slack](https://slack.cncf.io/). We also ran into some other issues which would create replication loops causing a crash as well. We addressed this issue by moving to a routing-ingesting receiver topology as suggested by the community.
 
 ## Cost Savings
 
 After concluding the POC we determined that it was feasible and we also found that the cost savings were pretty significant:
 
 ### M3DB Cost breakdown
 
-| Service       | Instance type   | Price      | Count | Total price (monthly) |
-|---------------|-----------------|------------|-------|-----------------------|
-| m3db          | n2d-highmem-32  | $1,172.61  | 18    | $21,106.98            |
-| m3coordinator | n2d-standard-48 | $1,303.73  | 9     | $11,733.57            |
-| M3 TOTAL      |                 | $32,840.55 |       |                       |
+<table class="table">
+    <thead>
+        <tr>
+            <th>Service</th>
+            <th>Instance type</th>
+            <th>Price</th>
+            <th>Count</th>
+            <th>Total price (monthly)</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>m3db</td>
+            <td>n2d-highmem-32</td>
+            <td>$1,172.61</td>
+            <td>18</td>
+            <td>$21,106.98</td>
+        </tr>
+        <tr>
+            <td>m3coordinator</td>
+            <td>n2d-standard-48</td>
+            <td>$1,303.73</td>
+            <td>9</td>
+            <td>$11,733.57</td>
+        </tr>
+        <tr>
+            <td>M3 TOTAL</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>$32,840.55</td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
I do not know how this will render from .md to the publishing engine. I am not sure I would make this change.

Reply:
I tested it locally, it renders correctly. Hugo has a setting to allow HTML, and [it's on](https://github.com/thanos-io/thanos/blob/2339836309ff50c235ab91581fbf4c7c6a1f6a10/website/hugo.yaml#L72). The problem with using markdown native tables is that they render to correct HTML, but the website uses Bootstrap, and [it's tables are opt-in](https://getbootstrap.com/docs/5.3/content/tables/). So if you don't add a `.table` class, only the reset styles apply, removing all padding and lines. Hugo's markdown rendering engine has a feature to add CSS classes to block elements, but I couldn't make it work, so I choose to put HTML here. Another option would be to add custom CSS to add styles for all tables under `.blog-post`.

Reply:
I'm fine with the change if you tested it.